README.md

```diff
@@ -33,4 +45,3 @@ Lombok has a lot of useful annotations that give you some of the nice things in
 ### Efficient Primitive Collections
```

Comment:
Here's a [slideshow](https://www.jcp.org/aboutJava/communityprocess/ec-public/materials/2014-05-1314/GS_Collections_May2014.pdf) that has some charts and comparisons.